src/index.html

```diff
@@ -1,13 +1,415 @@
 <!DOCTYPE html>
-<html lang="en">
+<html class="page" lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <link rel="icon"
+          type="image/svg+xml"
+          href="./images/icons/logo-favicon.svg"
+    >
+    <title>Museum</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+
+  <body class="page__body">
+    <header class="header" id="header">
+      <div class="top-bar header__top">
+        <a href="#" class="top-bar__logo-link">
+          <img src="./images/icons/logo.svg"
+               alt="NAMU logo"
+               class="top-bar__logo"
+          >
```

Comment:
Need to fix code style everywhere
```suggestion
          <img
            src="./images/icons/logo.svg"
            alt="NAMU logo"
            class="top-bar__logo"
          >
```